itou/templates/apply/list_for_siae.html

```diff
@@ -29,6 +30,32 @@ <h2>{{ siae.display_name }}</h2>
     {% endif %}
 {% endblock %}
 
+{% block messages %}
+    {{ block.super }}
+    {% if request.current_organization.is_subject_to_eligibility_rules %}
+        <div id="siae-employers-survey-about-autonomous-candidates" class="alert alert-important alert-dismissible-once d-none fade show" role="status">
```

Comment:
Idem que sur l’autre PR.
```suggestion
        <div id="siae-employers-survey-about-autonomous-candidates" class="alert alert-important alert-dismissible-once d-none" role="status">
```